app/javascript/components/CheckIn/CheckIn.jsx

```diff
@@ -21,22 +22,23 @@ const Checkin = ({ authenticity_token, checkIn, checkOut }) => {
     }
     )
   }
+
   const checkOutEmployee = secret => {
     axios.post('/check-out', {
       private_number: secret
     }).then((res) => {
-      const { message } = res.data[0];
-      setSuccessMessage(message)
+      const { message } = res.data;
+      setSuccessMessage(message[0]);
```

Comment:
you can use `message.first` to get the first element

Reply:
Right we can do that, we add it in a next PR. Thanks